honestcash-v2/src/app/shared/components/avatar/avatar.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+             selector: 'app-avatar',
+             template: `<img class="avatar" [ngClass]="{'large': large, 'small': !large}" [src]="src"/>`,
+             styles: [
```

Comment:
Please extract to a separate files.

Reply:
you should comment when things are refactored